recipes/install.rb

```diff
@@ -48,3 +47,11 @@
     cwd     "#{node[:valhalla][:src_dir]}/#{repo}"
   end
 end
+
+# link the lua transforms from the checkout
+%w(vertices edges admins).each do |lua|
```

Comment:
the reason i dont like it here is that this recipe is in a separte repo than the code right. which means that it has to just guess at where these lua files are. the script on the other hand lives right along side the lua files in the same repo. so it doesnt have to guess it knows, or is more likely to be in sync than this recipe